torchtune/modules/tokenizer.py

```diff
@@ -60,24 +62,33 @@ def encode(
         text: str,
         add_bos: bool = True,
         add_eos: bool = True,
+        trim_leading_whitespace: bool = False,
     ) -> List[int]:
         """Encode text into token IDs.
 
         Args:
             text (str): The input text to be encoded, unbatched.
             add_bos (bool): Whether to prepend BOS to the input, defaults to True.
             add_eos (bool): Whether to append EOS to the input, defaults to True.
-
+            trim_leading_whitespace (bool): Whether to trim leading whitespace from
```

Comment:
wanna link the discussion you found here so people know why this is a thing?

Reply:
On second thought, I think it might be a bit misleading to just link that with no context. I am gonna add my own comment that imo is more relevant.